src/__tests__/median.test.js

```diff
@@ -1,11 +1,39 @@
 import { median } from '../median.js';
 
 describe('median', () => {
-  it('should return 42', () => {
+  it('four elements having unit high, should return 2.5', () => {
     const data = {
       x: [1, 2, 3, 4],
-      y: [2, 2, 1, 1],
+    y: [1, 1, 1, 1],
     };
-    expect(median(data)).toBeCloseTo(13 / 6, 10);
+    expect(median(data)).toStrictEqual(2.5);
+  });
+  it('should return 2', () => {
+    const data = {
+      x: [1, 2, 3],
+      y: [1, 2, 1],
+    };
+    expect(median(data)).toStrictEqual(2);
+  });
+  it('should return 1', () => {
+    const data = {
+      x: [1, 2],
+      y: [3, 1],
+    };
+    expect(median(data)).toStrictEqual(1);
+  });
+  it('should return 2.5', () => {
```

Comment:
Please use more comprehensible message and not just a message that gives the expected result.

For example 'check even number of points' or 'check middle point'